cargo-dist/tests/integration-tests.rs

```diff
@@ -363,6 +363,7 @@ targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-pc-windows
 ci = ["github"]
 build-local-artifacts = false
 local-artifacts-jobs = ["./local-artifacts"]
+github-custom-job-permissions = {{ local-artifacts = ["packages: write"] }}
```

Comment:
example of using the feature (note the `{`'s are doubled because this is a rust format)

Reply:
appreciate pointing this out- would be good to get docs on this feature otherwise i am eager to unfork them and we have config 1.0 coming up so i say ship it